From 287488421a3df08bb7aa58fe5ab950c529c878e1 Mon Sep 17 00:00:00 2001
From: Prarit Bhargava <prarit@redhat.com>
Date: Thu, 28 Aug 2014 14:24:29 -0400
Subject: [cpufreq] intel_pstate: Remove unnecessary intermediate variable sample_time

Message-id: <1409235879-1283-176-git-send-email-prarit@redhat.com>
Patchwork-id: 89462
O-Subject: [RHEL7.1 PATCH BZ 1134639 175/185] cpufreq: intel_pstate: Remove unnecessary intermediate variable sample_time
Bugzilla: 1134369
RH-Acked-by: Stefan Assmann <sassmann@redhat.com>
RH-Acked-by: Mark Langsdorf <mlangsdo@redhat.com>
RH-Acked-by: Tony Camuso <tcamuso@redhat.com>

Bugzilla: https://bugzilla.redhat.com/show_bug.cgi?id=1134639

commit abf013bffeb55744fb6d1ff9bb30acaabe6302ab
Author: Stratos Karafotis <stratosk@semaphore.gr>
Date:   Fri Jul 18 08:37:22 2014 -0700

    cpufreq: intel_pstate: Remove unnecessary intermediate variable sample_time

    Remove the unnecessary intermediate assignment and use directly the
    pid_params.sample_rate_ms variable.

    Signed-off-by: Stratos Karafotis <stratosk@semaphore.gr>
    Signed-off-by: Dirk Brandewie <dirk.j.brandewie@intel.com>
    Signed-off-by: Rafael J. Wysocki <rafael.j.wysocki@intel.com>

Cc: Lenny Szubowicz <lszubowi@redhat.com>
Cc: Tony Camuso <tcamuso@redhat.com>
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/drivers/cpufreq/intel_pstate.c b/drivers/cpufreq/intel_pstate.c
index 18f3e6f..a998bd8 100644
--- a/drivers/cpufreq/intel_pstate.c
+++ b/drivers/cpufreq/intel_pstate.c
@@ -602,10 +602,9 @@ static inline void intel_pstate_sample(struct cpudata *cpu)
 
 static inline void intel_pstate_set_sample_time(struct cpudata *cpu)
 {
- int sample_time, delay;
+ int delay;
 
- sample_time = pid_params.sample_rate_ms;
- delay = msecs_to_jiffies(sample_time);
+ delay = msecs_to_jiffies(pid_params.sample_rate_ms);
  mod_timer_pinned(&cpu->timer, jiffies + delay);
 }
 
-- 
1.7.1